app/logic/emailHandler.py

```diff
@@ -126,7 +129,29 @@ def send(self, message: Message):
     # The email templates are stored inside of the emailHandler model, and depending on which email template
     # is pulled from the model, and replaceText method will replace the neccesary keywords with the correct data.
     # The sendEmail method will handle all of the email sending once the email template has been populated.
-    def laborStatusFormSubmitted(self):
+    def laborStatusFormSubmitted(self, studentEmailPurpose=False):
+        try:
+            self.student = Student.get(Student.ID == str(self.laborStatusForm.studentSupervisee.ID))
+            self.studentEmail = self.student.STU_EMAIL
+
+            # generating a link for confirmation for student
+            self.confirmationLink = f"{request.host_url}studentResponse/confirm?token={self.laborStatusForm.confirmationToken}"
+
+            emailTemplate = EmailTemplate.get(EmailTemplate.purpose == "Labor Status Form Submitted For Student")
+            emailBody = emailTemplate.body
+
+            emailBody = emailBody.replace("@@ConfirmationLink@@", self.confirmationLink)
+
+            if not studentEmailPurpose:  
+                message = Message(subject=emailTemplate.subject, recipients=[self.studentEmail])
+                message.html = emailBody
+                self.send(message)
```

Comment:
These methods don't usually send an email, do they? Are we sure we need to do that here?